pom.xml

```diff
@@ -27,7 +27,7 @@
   </parent>
   <groupId>io.fabric8.jenkins.plugins</groupId>
   <artifactId>openshift-sync</artifactId>
-  <version>1.0.40-SNAPSHOT</version>
+  <version>1.0.41-SNAPSHOT</version>
```

Comment:
You should rebase this PR to pick up https://github.com/openshift/jenkins-sync-plugin/commit/1baa73027d14452da976f60050c78afeed3d6c80#diff-600376dffeb79835ede4a0b285078036 from https://github.com/openshift/jenkins-sync-plugin/pull/322 and not "re-introduce" this change with this PR.